core/src/main/java/io/grpc/PickFirstLoadBalancerFactory.java

```diff
@@ -42,30 +42,29 @@
 import javax.annotation.concurrent.GuardedBy;
 
 /**
- * A {@link LoadBalancer} that provides simple round-robin and pick-first routing mechanism over the
+ * A {@link LoadBalancer} that provides pick-first routing mechanism over the
  * addresses from the {@link NameResolver}.
  */
-// TODO(zhangkun83): Only pick-first is implemented. We need to implement round-robin.
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1771")
-public final class SimpleLoadBalancerFactory extends LoadBalancer.Factory {
+public final class PickFirstLoadBalancerFactory extends LoadBalancer.Factory {
```

Comment:
I think it's inaccurate to name a class based on a feature that is completely provided by another class. A proper name for this class should convey that it doesn't do load-balancing at all. "DummyLoadBalancer" sounds better to me, but I will defer to you native speakers.